src/libcrun/cgroup-utils.c

```diff
@@ -742,10 +807,23 @@ write_controller_file (const char *path, int controllers_to_enable, libcrun_erro
               ret = write_file (subtree_control, token, strlen (token), err);
               if (ret < 0)
                 {
-                  if (crun_error_get_errno (err) == EBUSY)
-                    repeat = true;
+                  e = crun_error_get_errno (err);
                   crun_error_release (err);
 
+                  if (e == EOPNOTSUPP)
+                    {
+                      make_cgroup_threaded (path, err);
```

Comment:
Not sure if this is intentional but  `make_cgroup_threaded` called twice once here and again just below this.
```suggestion
                      
```

Reply:
yes, this is a mistake, I'll fix it